app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -1,3 +1,6 @@
+<%# TODO: set these variables based on the current user %>
```

Comment:
I don't get what these roles mean here. So these will be replaced with an ability check or what?